docs/pages/includes/config-reference/app-service.yaml

```
@@ -5,6 +5,17 @@ app_service:
     # Application Service is working correctly. The app outputs JWTs so it can
     # be useful when extending your application.
     debug_app: true
+
+    # Matchers for application resources created with "tctl create" command
```

```suggestion
    # Matchers for dynamic application resources
```
Since `tctl create` isn't the only way to create dynamic resources